frontend/test/e2e/auth/login.spec.js

```diff
@@ -25,22 +25,22 @@ describeE2E("auth/login", () => {
             await driver.manage().deleteAllCookies();
         });
 
-        it ("should take you to the login page", async () => {
+        it("should take you to the login page", async () => {
             await driver.get(`${server.host}/`);
             await waitForUrl(driver, `${server.host}/auth/login?redirect=%2F`);
             expect(await driver.isElementPresent(By.css("[name=email]"))).toEqual(true);
             await screenshot(driver, "screenshots/auth-login.png");
         });
 
-        it ("should log you in", async () => {
+        it("should log you in", async () => {
             await driver.get(`${server.host}/`);
             await loginMetabase(driver, "bob@metabase.com", "12341234");
             await waitForUrl(driver, `${server.host}/`);
             const sessionCookie = await driver.manage().getCookie("metabase.SESSION_ID");
             sessionId = sessionCookie.value;
         });
 
-        xit ("should redirect you after logging in", async () => {
+        it("should redirect you after logging in", async () => {
```

Comment:
TDD brah.

I can't remember when I originally added it but there was a separate PR for this awhile ago, but I merged the E2E stuff into it and it became a mess so I opened a new one.